src/api/app/views/webui2/webui/request/_show_overview.html.haml

```diff
@@ -1,9 +1,11 @@
 %h3
-  %p Request #{bs_request.number} (#{bs_request.state})
+  Request #{bs_request.number} (#{bs_request.state})
 
 #description-text
   - if bs_request.description.present?
-    = render partial: 'webui/webui/collapsible_text', locals: { text: bs_request.description, threshold: { text_length: 1000, line_count: 10 } }
+    = render partial: 'webui/webui/collapsible_text', locals: { text: bs_request.description,
+                                                                threshold: { text_length: 1000, line_count: 10 },
```

Comment:
you pass the line_count as paramether here, but the css doesn't have this into account, right?

Reply:
right, it's only for the [partial](https://github.com/openSUSE/open-build-service/pull/7411/files#diff-7f2502bfced4ac1e5ad0cf3307db01b4R4).

Reply:
I know, but if you use another number here it will break the css.

Reply:
We talked offline and we decide to use the `line_count` as class instead of passing `type` as a local variable.